ibis/backends/postgres/registry.py

```diff
@@ -710,7 +710,7 @@ def _day_of_week_name(t, expr):
         ops.StandardDev: variance_reduction('stddev'),
         ops.RandomScalar: _random,
         # now is in the timezone of the server, but we want UTC
-        ops.TimestampNow: lambda *args: sa.func.timezone('UTC', sa.func.now()),
+        ops.TimestampNow: lambda *_: sa.func.timezone('UTC', sa.func.now()),
```

Comment:
Perhaps it deserves a named function `_now`?

Reply:
Seems simple enough to leave it as a `lambda` for now.